src/etc/inc/interfaces.inc

```diff
@@ -3929,7 +3959,13 @@ function interface_dhcpv6_configure($interface = "wan", $wancfg) {
 
 			$dhcp6cconf .= "\trequest domain-name-servers;\n";
 			$dhcp6cconf .= "\trequest domain-name;\n";
+			
+			/* dhcp6c will run different scripts depending on whether dhcpwithoutra is set or unset. */
+			if (isset($wancfg['dhcp6withoutra'])) {
+			$dhcp6cconf .= "\tscript \"{$g['varetc_path']}/dhcp6c_{$interface}_dhcp6withoutra_script.sh\"; # we'd like nameservers and RTSOLD to do all the work\n";
```

Comment:
Yes as two scripts are used, one called by dhcp6 ( dhcp6_*_withoutra_script.sh ) and RTSOLD which calls ( dhcp6_*_script.sh) the later being reused, otherwise there would be three scripts.